tests/end2end/features/zoom.feature

```diff
@@ -81,6 +81,16 @@ Feature: Zooming in and out
         Then the message "Zoom level: 60%" should be shown
         And the zoom should be 60%
 
+    # https://github.com/qutebrowser/qutebrowser/issues/2507
+    Scenario: Qutebrowser enforces correct zoom level
+        When I run :zoom 150%
+        And I open data/search.html
+        And I run :open http://127.0.0.1:(port)/data/long_load.html
```

Comment:
Please add a comment here saying that loading 127.0.0.1 is required for this test to work.